stock_picking_show_return/models/stock_picking.py

```diff
@@ -10,9 +11,28 @@ class StockPicking(models.Model):
     returned_ids = fields.Many2many(
         comodel_name="stock.picking", compute="_compute_returned_ids",
         string="Returned pickings")
+    origin_returned_id = fields.Many2one(
+        comodel_name="stock.picking",
+        compute="_compute_origin_returned_id",
+        string="Origin picking")
 
     @api.multi
     def _compute_returned_ids(self):
         for picking in self:
             picking.returned_ids = picking.mapped(
                 'move_lines.returned_move_ids.picking_id')
+
+    @api.multi
```

Comment:
```suggestion
    @api.depends('move_lines.origin_returned_move_id')
```

Reply:
The field is compute no store...

Reply:
It doesn't matter being stored or not. Cached values takes `depends` as the rule for updating local value or not, so always put depends.